charts/apisix-dashboard/README.md

```diff
@@ -67,17 +67,22 @@ The following tables lists the configurable parameters of the apisix-dashboard c
-| `config.authentication.expireTime`                  | JWT token expire time, in second | `3600` |
-| `config.authentication.users`                  | Specifies username and password for login `manager api`. | `[{username: admin, password: admin}]` |
-| `config.conf.etcd.endpoints`                       | Supports defining multiple etcd host addresses for an etcd cluster                                                                  | `apisix-etcd:2379`             |
-| `config.conf.etcd.prefix`                       | Apache APISIX config's prefix in etcd, /apisix by default                                                                  | `/apisix`             |
-| `config.conf.etcd.username`                       | Specifies etcd basic auth username if  enable etcd auth                                                                | `~`             |
-| `config.conf.etcd.password`                       | Specifies etcd basic auth password  if  enable etcd auth                                                              | `~`             |
-| `config.conf.log.accessLog.filePath`                  | Access log path | `/dev/stdout` |
-| `config.conf.log.errorLog.filePath`                  | Error log path | `/dev/stderr` |
-| `config.conf.log.errorLog.level`                  | Error log level. Supports levels, lower to higher: debug, info, warn, error, panic, fatal | `warn` |
+| `config.authentication.expireTime`              | JWT token expire time, in second | `3600` |
+| `config.authentication.users`                   | Specifies username and password for login `manager api`. | `[{username: admin, password: admin}]` |
+| `config.conf.etcd.endpoints`                    | Supports defining multiple etcd host addresses for an etcd cluster                                                    | `apisix-etcd:2379` |
+| `config.conf.etcd.prefix`                       | Apache APISIX config's prefix in etcd, /apisix by default                                                             | `/apisix`       |
+| `config.conf.etcd.username`                     | Specifies etcd basic auth username if  enable etcd auth                                                               | `~`             |
+| `config.conf.etcd.password`                     | Specifies etcd basic auth password  if  enable etcd auth                                                              | `~`             |
+| `config.conf.etcd.mtls.enabled`                 | Enable MTLS option when connecting to an etcd cluster with TLS enabled                                                | `false`         |
+| `config.conf.etcd.mtls.existingSecret`          | Name of existing secret that contains certs for TLS authentication                                                    | `""`            |
+| `config.conf.etcd.mtls.caFile`                  | Key of CA file in secret                                                                                              | `ca.crt`        |
```

Comment:
This field will also be required even if we don't enable mTLS, i.e., just the simple TLS. I prefer to call the field `mtls` to `tls`.

Reply:
LGTM. then should this [line](https://github.com/apache/apisix-dashboard/blob/master/api/internal/conf/conf.go#L86) be modified as well?